jina/peapods/runtimes/gateway/prefetch.py

```diff
@@ -16,6 +17,21 @@
 class PrefetchMixin(ABC):
     """JinaRPCServicer """
 
+    def __init__(self):
```

Comment:
a mixin should not have `state` if possible. This should be some kind of abstract property

Reply:
I removed the mixin now as its no longer possible actually to have this as a mixin I think. I adapted the class hierarchy slightly to reflect this.